tests/test_compat.py

```diff
@@ -0,0 +1,69 @@
+from django.test import TestCase
+
+from rest_framework import compat
+
+
+class CompatTests(TestCase):
+
+    def test_total_seconds(self):
+        class MockTimedelta(object):
+            days = 1
+            seconds = 1
+            microseconds = 100
+        timedelta = MockTimedelta()
+        expected = (timedelta.days * 86400.0) + float(timedelta.seconds) + (timedelta.microseconds / 1000000.0)
```

Comment:
Is there any reason why we chose to calculate the expected value in the test instead of working from a known-good value for how many seconds there are?